CommandLineClient/src/main/java/com/dslplatform/compiler/client/parameters/DslCompiler.java

```diff
@@ -793,11 +793,11 @@ private static void downloadCompiler(
 
 	private static boolean testCompiler(final Context context, final File path) throws ExitException {
 		if (Utils.isWindows()) {
-			return Utils.testCommand(context, path.getAbsolutePath(), "DSL Platform");
+			return Utils.testCommand(context, path.getAbsolutePath(), "DSL Platform", Charset.forName("UTF-8"));
```

Comment:
Test command is not parsing custom output of the compiler. This not really required

Reply:
Undided it.